### PR TITLE
Update the Network Sensor Deployment Guide to clarify firewall rules and remove references to legacy CSE servers

### PR DESCRIPTION
**Pull request by josh-williams**
## Purpose of this pull request

This pull request (PR) makes the Network Sensor Deployment Guide clearer by removing a bunch of relevant that was only relevant to the legacy CSE server setup that is no longer in use for any customers.

Issue number: TRIDENT-16931

## Select the type of change
- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/cse/sensors/network-sensor-deployment-guide.md [hunk 1]
```diff
@@ -320,21 +295,13 @@ Description. The directory to which the sensor writes files extracted from netwo
 
 ### filter
 
-**Description.** This option tells bro not to capture network traffic between the sensor and the Sumo Logic (or the CSE server, in legacy environments). The sensor sends status reports and bro output files to end points on the destination, which we don’t want bro to capture. 
+**Description.** This option tells bro not to capture network traffic between the sensor and the Sumo Logic. The sensor sends status reports and bro output files to end points on the destination, which we don’t want bro to capture.
```
Comment (kimsauce) [inline docs/cse/sensors/network-sensor-deployment-guide.md:298]: @josh-williams @jpipkin1 double checking that `bro` is the correct terminology here!

**Review by kimsauce — COMMENTED** [review]
one question

File: docs/cse/sensors/network-sensor-deployment-guide.md (hunk 1 above)
Comment (jpipkin1) [reply docs/cse/sensors/network-sensor-deployment-guide.md:298]: @josh-williams, you should change instances of "bro" to "Bro" when you are talking about the application. @kimsauce, I looked it up, and found that Bro is the older name of an open-source network traffic analyzer, which is now known as Zeek.

File: docs/cse/sensors/network-sensor-deployment-guide.md [hunk 2]
```diff
@@ -252,22 +243,6 @@ If you do make manual updates to `trident-sensor.cfg`, you must restart the Netw
...
-
-**Description**. The address of a CSE legacy server.
-
-**Default Value.** None
-
-**Configured by wizard?** Yes, it is set based on the cluster name provided in the wizard dialog when it prompts you "Enter the address of your Cloud SIEM cluster". This prompt appears only if you do not supply a value when prompted for a Sumo Logic HTTP Source URL.
-
 ### compression
 
 **Description.** This option controls whether the sensor compresses the bro output files stored in ` /opt/trident/sensor/output/`. By default the sensor does compress the bro files using gzip, when the size of the bro file exceeds the value of the `compression_threshold`. option, described below. To turn compression off, set this option to “no”, or any value other than “gzip”.  Compresses when the parameter is missing or set to gzip and bro output file is above the value set in `compression_threshold`.
```
Comment (jpipkin1) [inline docs/cse/sensors/network-sensor-deployment-guide.md:248]:
```suggestion
**Description.** This option controls whether the sensor compresses the Bro output files stored in ` /opt/trident/sensor/output/`. By default, the sensor does compress the Bro files using gzip when the size of the Bro file exceeds the value of the `compression_threshold` option, described below. To turn compression off, set this option to “no”, or any value other than “gzip”.  Compression occurs when the parameter is missing or set to gzip and the Bro output file is above the value set in `compression_threshold`.
```

File: docs/cse/sensors/network-sensor-deployment-guide.md (hunk 1 above)
Comment (jpipkin1) [inline docs/cse/sensors/network-sensor-deployment-guide.md:298]:
```suggestion
**Description.** This option tells Bro not to capture network traffic between the sensor and the Sumo Logic. The sensor sends status reports and Bro output files to end points on the destination, which we don’t want Bro to capture. 
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@josh-williams, it looks good. I flagged a few places where you need to change "bro" to "Bro".

File: docs/cse/sensors/network-sensor-deployment-guide.md (hunk 1 above)
Comment (jpipkin1) [reply docs/cse/sensors/network-sensor-deployment-guide.md]: Fixed.

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**